src/lex-keyword.cpp

```diff
@@ -1,3 +1,6 @@
+/* Code generated by tools/generate-lex-keyword. DO NOT EDIT. */
+/* source: src/lex-keyword.gperf */
```

Comment:
Why does this file list a 'source', but other files don't?

Reply:
Because I do know the source files for most of them. i will try to add rest of them